docs/src/main/paradox/docs/delta/api/views/elasticsearch-view-api.md

```diff
@@ -11,15 +11,43 @@ A default view gets automatically created when the project is created but other
 
 An asynchronous process gets triggered for every view. This process can be visualized as a pipeline with different stages.
 
-The first stage is the input of the pipeline: a stream of events scoped for the project where the view was created.
+The first stage is the input of the pipeline: a stream of resource updates scoped for the project where the view was created.
 
-For each event:
+For each incoming update, a pipeline is applied to this resource.
 
-* The related resource is fetched
-* A series of filters and transformations is then applied to this resource
-* If the resource is filtered out, the previous version of the resource is removed from the index if present
-* If the resource makes it to the end of the pipeline, a Json document is generated by merging the original payload, 
-the data and metadata graphs and then stored in the corresponding Elasticsearch index.
+A pipeline is made of pipes which are the processing units of a pipeline for an Elasticsearch view.
+
+They are applied sequentially as defined by the user in the view payload and allow to transform and/or filter a resource
+before indexing it to Elasticsearch.
+
+### Filters
+
+Filters allow to discard a resource if it does not match a given predicate (ex: it has been deprecated or does not match an expected type).
+If a resource is filtered out by a pipe:
+
+* it won't be indexed so the execution and it will be deleted from the index if a previous revision of this resource has been indexed
+* the execution of the next pipes is short-circuited to avoid useless computation
+
+It is therefore encouraged to apply the filtering pipes at the beginning of the pipeline.
+
+Existing pipes and how to create new ones is defined @ref:[here](./pipes.md).
+
+### Transformations
+
+Transformations allow to modify:
+
+* The original payload of the resource which has been submitted by the user
+* The metadata graph of the resource (the fields that have been generated from Nexus)
+* The data graph of the resource (that has been computed from the original payload at ingestion time)
+
+### Reconciliation and indexing
+
+When a resource makes it to the end of the pipeline, we produce a json document by merging:
+
+* the original payload,
+* the data and metadata graphs that have been translated to the compacted JSON-LD format with a default context and the one provided by the view
```

Comment:
Maybe adding a link to the default context?

Reply:
The thing is that there are two of them that we merge at runtime: 
https://github.com/BlueBrain/nexus/blob/c85a0bc056d6e6833c8df93ef9020280fa7df96f/delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/indexing/IndexingViewDef.scala#L37

Reply:
Ok, let's see then if there are any questions about it later